### DISCUSSION TITLE
DPI/Resolution/Zoom issue

### DISCUSSION THREAD

**Discussion by mgcarnevali** [kangyu-california/PersistentWindows#349] [General]
This problem started to happen again:

https://github.com/kangyu-california/PersistentWindows/discussions/306

**Comment by kangyu-california** [comment]
Please follow the 5.54 release note to modify the DPI setting of PersistentWindows.exe

**Comment by kangyu-california** [comment]
Starting from 5.55, the DPI setting fix has been integrated into auto-start pw task script.

**Reply by mgcarnevali** [reply]
I'm using version 5.58, should i modify the dpi setting anyway? persistentwindows.exe starts with windows using the vbs file

**Reply by kangyu-california** [reply]
Yes, manually modify the dpi setting (detailed in 5.54 release note) is recommended in your case

**Reply by mgcarnevali** [reply]
unfortunately auto_start_pw.bat is not working, at least in my system...

D:\Descargas\PersistentWindows>auto_start_pw.bat

D:\Descargas\PersistentWindows>powershell -ExecutionPolicy Bypass -File "D:\Descargas\PersistentWindows\auto_start_pw_aux.ps1"
Register-ScheduledTask : El parámetro no es correcto.
(9,21):UserId:Max
En D:\Descargas\PersistentWindows\auto_start_pw_aux.ps1: 25 Carácter: 1
+ Register-ScheduledTask -TaskName $taskName -Action $action -Trigger $ ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (PS_ScheduledTask:Root/Microsoft/...S_ScheduledTask) [Register-Schedule
   dTask], CimException
    + FullyQualifiedErrorId : HRESULT 0x80070057,Register-ScheduledTask

Get-ScheduledTask : No se encontraron objetos MSFT_ScheduledTask cuya propiedad 'TaskName' sea igual a
'StartPersistentWindowsMax'. Compruebe el valor de la propiedad e inténtelo de nuevo.
En D:\Descargas\PersistentWindows\auto_start_pw_aux.ps1: 27 Carácter: 9
+ $task = Get-ScheduledTask -TaskName $taskName
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (StartPersistentWindowsMax:String) [Get-ScheduledTask], CimJobException
    + FullyQualifiedErrorId : CmdletizationQuery_NotFound_TaskName,Get-ScheduledTask

La propiedad 'ExecutionTimeLimit' no se encuentra en este objeto. Compruebe que la propiedad existe y se puede
establecer.
En D:\Descargas\PersistentWindows\auto_start_pw_aux.ps1: 29 Carácter: 1
+ $taskSettings.ExecutionTimeLimit = "PT0S" # disable time limit
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : PropertyNotFound

Set-ScheduledTask : No se puede validar el argumento del parámetro 'Settings'. El argumento es null o está vacío.
Proporcione un argumento que no sea null o que no esté vacío e intente ejecutar el comando de nuevo.
En D:\Descargas\PersistentWindows\auto_start_pw_aux.ps1: 30 Carácter: 49
+ Set-ScheduledTask -TaskName $taskName -Settings $taskSettings
+                                                 ~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Set-ScheduledTask], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Set-ScheduledTask

No se puede indizar en una matriz nula.
En D:\Descargas\PersistentWindows\auto_start_pw_aux.ps1: 32 Carácter: 1
+ $task.Actions[0].Arguments = $arguments
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : NullArray

Set-ScheduledTask : No se puede validar el argumento del parámetro 'TaskPath'. El argumento es null o está vacío.
Proporcione un argumento que no sea null o que no esté vacío e intente ejecutar el comando de nuevo.
En D:\Descargas\PersistentWindows\auto_start_pw_aux.ps1: 33 Carácter: 49
+ ... et-ScheduledTask -TaskName $taskName -TaskPath $task.TaskPath -Action ...
+                                                    ~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Set-ScheduledTask], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Set-ScheduledTask

La propiedad 'Principal' no se encuentra en este objeto. Compruebe que la propiedad existe y se puede establecer.
En D:\Descargas\PersistentWindows\auto_start_pw_aux.ps1: 37 Carácter: 1
+ $task.Principal = $principal
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : PropertyNotFound

Set-ScheduledTask : No se puede validar el argumento del parámetro 'TaskPath'. El argumento es null o está vacío.
Proporcione un argumento que no sea null o que no esté vacío e intente ejecutar el comando de nuevo.
En D:\Descargas\PersistentWindows\auto_start_pw_aux.ps1: 38 Carácter: 49
+ ... et-ScheduledTask -TaskName $taskName -TaskPath $task.TaskPath -Princi ...
+                                                    ~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Set-ScheduledTask], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Set-ScheduledTask

**Reply by kangyu-california** [reply]
This is a known issue #284 due to bug in Windows OS.

**Reply by mgcarnevali** [reply]
I used Task Scheduler (Windows 7/10/11) method and it worked, is it the same than the auto_start_pw.bat method?

**Reply by kangyu-california** [reply]
Those two methods are effectively the same

**Reply by mgcarnevali** [reply]
DPI/Zoom issue still persists even using task scheduler method

![steam](https://github.com/user-attachments/assets/dfaf8f68-26f3-428d-b0ba-fb08cd026830)

**Comment by mgcarnevali** [comment]
Hello,

I could use the auto_start_pw.bat file after changing the computer name but as you can see in the attached screenshot the issue still persists in Steam.

I took the picture with my phone so you can see my entire monitor to confirm it.
![steam](https://github.com/user-attachments/assets/d7e2b270-30ce-4cfa-98c7-e89579fcc7a7)

Any idea how to fix it?

**Reply by kangyu-california** [reply]
Pw does not control the font zoom behavior. You should raise the issue to Steam.
